modules/end-user-guide/partials/proc_building-an-image-for-an-ide-based-on-the-intellij-platform.adoc

```diff
@@ -0,0 +1,177 @@
+
+** link:https://stedolan.github.io/jq/[jq]
+
+
+.Procedure
+
+. Get a local copy of the link:https://github.com/che-incubator/jetbrains-editor-images[JetBrains Projector Editor Images repository].
++
+----
+$ git clone https://github.com/che-incubator/jetbrains-editor-images
+$ cd jetbrains-editor-images
+----
+
+. Run the build script with parameters:
++
+[subs="+quotes,macros,attributes"]
+----
+$ ./projector build --tag __<tag>__ --url __<URL>__
+----
++
+`--tag __<tag>__`::
+The name and tag to apply to the image after build in `name:tag` format.
++
+`--url __<url>__`::
+The URL pointing to an archive of the IDE based on the link:https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html#intellij-platform-based-products-of-recent-ide-versions[IntelliJ Platform version 2020.3]. The archive must target the Linux platform, be in `+tar.gz+` format, and include JetBrains Runtime (JBR). 
++
+.Building the image with IntelliJ IDEA Community 2020.3.3
+====
+----
+./projector.sh build --tag che-idea:2020.3.3 \
+  --url https://download-cdn.jetbrains.com/idea/ideaIC-2020.3.3.tar.gz
+----
+====
++
+.Building the image with PyCharm Community 2020.3.5
+====
+----
+$ ./projector.sh build --tag che-pycharm:2020.3.5 \
+  --url https://download.jetbrains.com/python/pycharm-community-2020.3.5.tar.gz
+----
+====
++
+.Building the image with WebStorm 2020.3.3
+====
+----
+$ ./projector.sh build --tag che-webstorm:2020.3.3 \
+  --url https://download.jetbrains.com/webstorm/WebStorm-2020.3.3.tar.gz
+----
+====
++
+.Building the image with IntelliJ IDEA Ultimate 2020.2.2
+====
+----
+$ ./projector.sh build --tag che-idea-ultimate:2020.2.2 \
+  --url https://download.jetbrains.com/idea/ideaIU-2020.2.2.tar.gz
+----
+====
++
+.Building the image with Android Studio 4.2.0.22
+====
+----
+$ ./projector.sh build --tag che-android-studio:4.2.0.22 \
+  --url https://redirector.gvt1.com/edgedl/android/studio/ide-zips/4.2.0.22/android-studio-ide-202.7188722-linux.tar.gz
+----
+====
+
+. To test the image, run it locally and go to `++http://localhost:8887++` to access the IDE.
++
+[subs="+quotes,macros,attributes"]
+----
+$ ./projector.sh run __<tag>__
+----
++
+.Testing the image with IntelliJ IDEA Community 2020.3.3
+====
+----
+$ ./projector.sh run che-idea:2020.3.3
+----
+====
++
+.Testing the image with PyCharm 2020.3.5
+====
+----
+$ ./projector.sh run che-pycharm:2020.3.5
+----
+====
++
+.Testing the image with WebStorm 2020.3.3
+====
+----
+$ ./projector.sh run che-webstorm:2020.3.3
+----
+====
++
+.Testing the image with IntelliJ IDEA Ultimate 2020.2.2
+====
+----
+$ ./projector.sh run che-idea-ultimate:2020.2.2
+----
+====
++
+.Testing the image with Android Studio 4.2.0.22
+====
+----
+$ ./projector.sh run che-android-studio:4.2.0.22
+----
+====
+
+. Publish the image to a registry accessible by {prod-short}, and remember the location: __<registry>/<image>:<tag>__.
```

Comment:
1. `remember the location` for when/which step?
2.  Suggestion `remember the location` `copy the URL`